libraries/Adafruit_LittleFS/src/littlefs/lfs_util.h

```diff
@@ -56,21 +56,21 @@ extern "C"
 // Logging functions
 #ifndef LFS_NO_DEBUG
 #define LFS_DEBUG(fmt, ...) \
-    printf("lfs debug:%d: " fmt "\n", __LINE__, __VA_ARGS__)
+    PRINTF("lfs debug:%d: " fmt "\n", __LINE__, __VA_ARGS__)
```

Comment:
@henrygab I got an local compilation issue with PRINTF is not defined when debug is enabled (which sneak through ci), it makes sense since lfs_util doens't include the `common_func.h` and it shouldn't though since it is an external library.

Reply:
#467 revert this change until fixed probably later on.